optimism/src/mips/constraints.rs

```diff
@@ -31,7 +32,7 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
     /// In the concrete implementation for the constraints, the interpreter will
     /// work over columns. The position in this case can be seen as a new
     /// variable/input of our circuit.
-    type Position = Column;
+    type Position = MIPSColumn;
```

Comment:
Note that within the interpreter, we still speak the ColumnAlias lingo. But then when expressions are returned, they are generic expressions over columns.